examples/train_xgboost/flow_gpu.py

```diff
@@ -0,0 +1,69 @@
+from metaflow import FlowSpec, step, pypi, kubernetes, environment, current, metaflow_ray
+from gpu_profile import gpu_profile
+
+
+class RayXGBoostGPU(FlowSpec):
+    n_files = 1500
+    data_url = "s3://obp-475b0e-metaflow/metaflow/mf.datasets/investment_ids"
```

Comment:
Make this into a parameter and remove this Bucket URL from here.

Reply:
done, added in readme though via `--data_url`